package.json

```diff
@@ -28,17 +29,19 @@
   },
   "peerDependencies": {
     "react": "^16.9",
-    "react-native": ">=0.60 <0.62"
+    "react-native": ">=0.60 <0.62",
+    "react-native-windows": "0.61.0-beta.27"
```

Comment:
This peer dep is too restrictive.  At least make it "^0.61.0-beta.27".   Or once we bring RNW 0.61 out of beta, it could match the react-native dependency version.

Reply:
Added "^" for now